src/it/xerces/src/test/java/com/jcabi/xerces/XercesSampleTest.java

```diff
@@ -56,6 +57,10 @@
      */
     @Test
     public void validatesXmlForSchemaValidity() throws Exception {
+        final int zero = 0;
```

Comment:
@yegor256 there are more numbers needed than just 0, 1 and -1. I've change them to eg. fiftytimes. What name would you suggest, if fifty and fiftytimes is not  a good name?